fastlane/lib/fastlane/actions/add_git_tag.rb

```diff
@@ -78,7 +78,7 @@ def self.available_options
                                        description: "The build number. Defaults to the result of increment_build_number if you\'re using it",
                                        default_value: Actions.lane_context[Actions::SharedValues::BUILD_NUMBER],
                                        default_value_dynamic: true,
-                                       is_string: false,
+                                       type: Integer,
```

Comment:
Earlier: `is_string: false` was supporting both type inputs `Integer + String`
Now: `type: Integer` will support only `Integer` input

Q: is it okay? or should we auto-convert value for `type: Integer` from string inside FastlaneCore::ConfigItem ?

Reply:
It looks like `default_value`is populated in `increment_build_number` as a String
https://github.com/fastlane/fastlane/blob/3456bdebc4a3089a0fd147f501c81557f74a5150/fastlane/lib/fastlane/actions/increment_build_number.rb#L51-L54

Reply:
I was thinking on adding an `Any` type for this cases which is just a placeholder for a parameter that can be an integer, a string, a double, a float or a symbol.

Reply:
> I was thinking on adding an `Any` type for this cases which is just a placeholder for a parameter that can be an integer, a string, a double, a float or a symbol.

Agree! this makes sense to me! 💯 

> So maybe this can wait a bit until this is implemented if no rush involved.

Agree! shall I keep using `is_string` **only for this param** for now? we can revisit again for such tricky params once we will have `Any` type support! what do you say @minuscorp?

Reply:
This can kind of be done with `skip_type_validation: true` 🤷‍♂️

Reply:
> This can kind of be done with `skip_type_validation: true` 🤷‍♂️

Ohh really, this is 💡 I will try that and keep you all posted in this thread! Thank you 🙇

Reply:
I can confirm, `skip_type_validation: true` works! YaY... 🎉